src/scripts/settings/webSettings.js

```diff
@@ -72,6 +72,10 @@ export function getIncludeCorsCredentials() {
 
 export function getMultiServer() {
     return getConfig().then(config => {
+        // Enable multi-server support when served by webpack
+        if (__WEBPACK_SERVE__) { // eslint-disable-line no-undef
+            return true;
+        }
         return !!config.multiserver;
```

Comment:
in case we have a bad config
```suggestion
    // Enable multi-server support when served by webpack
    if (__WEBPACK_SERVE__) { // eslint-disable-line no-undef
        return Promise.resolve(true);
    }

    return getConfig().then(config => {
        return !!config.multiserver;
```